docs/inputs.md

```diff
@@ -61,6 +61,16 @@
         <i>String &mdash; Default: "quay.io/biocontainers/biowdl-input-converter:0.2.1--py_0"</i><br />
         The docker image used for this task. Changing this may result in errors which the developers may choose not to address.
 </p>
+<p name="SubreadsProcessing.convertDockerImagesFile.memory">
+        <b>SubreadsProcessing.convertDockerImagesFile.memory</b><br />
+        <i>String &mdash; Default: "128M"</i><br />
+        The maximum aount of memroy the job will need.
```

Comment:
aount -> amount
memroy ->  memory

Reply:
Good catch, that is actually a typo in the parameter_meta of the common.wdl file in our tasks repository.